Cargo.toml

```
@@ -47,11 +47,14 @@ features = [
     "bevy_text",
     "bevy_sprite",
     #"file_watcher",
-    "multi-threaded",
+    "multi_threaded",
+    "webgl2",
+    "zstd",
```

```suggestion
    # TODO Remove these before release. See https://github.com/bevyengine/bevy/issues/13728
    "zstd",
```

Don't want this to get forgotten.